content/docs/start/list-of-operations.mdx

```diff
@@ -482,3 +488,116 @@ Possible errors:
+| CLAIM_CLAIMABLE_BALANCE_NO_TRUST | -4 | The source account does not trust the issuer of the asset it is trying to claim in the ClaimableBalanceEntry. |
+| CLAIM_CLAIMABLE_BALANCE_NOT_AUTHORIZED | -5 | The source account is not authorized to claim the asset in the ClaimableBalanceEntry. |
+
+## Begin Sponsoring Future Reserves
+
+Establishes the is-sponsoring-future-reserves-for relationship between the source account and sponsoredID. See link on how this operation should be used. TODO: Add link to sponsorship glossary page.
+
+There must be a corresponding [end sponsoring future reserves](#end-sponsoring-future-reserves) operation in the same transaction to end the is-sponsoring-future-reserves-for relationship. The transaction will fail otherwise.
+
+Threshold: Medium
+
+Result: `BeginSponsoringFutureReservesResult`
+
+| Parameters | Type | Description |
+| --- | --- | --- |
+| SponsoredID | account ID | Account that will have it's reserves sponsored. |
+
+Possible errors:
+
+| Error | Code | Description |
+| --- | --- | --- |
+| BEGIN_SPONSORING_FUTURE_RESERVES_MALFORMED | -1 | Source account is equal to sponsoredID. |
+| BEGIN_SPONSORING_FUTURE_RESERVES_ALREADY_SPONSORED | -2 | Source account is already sponsoring sponsoredID. |
+| BEGIN_SPONSORING_FUTURE_RESERVES_RECURSIVE | -3 | Either source account is currently being sponsored, or sponsoredID is sponsoring another account. |
+
+## End Sponsoring Future Reserves
+
+Terminates the current is-sponsoring-future-reserves-for relationship in which the source account is sponsored.
+
+Threshold: Medium
+
+Result: `EndSponsoringFutureReservesResult`
+
+| Error | Code | Description |
+| --- | --- | --- |
+| END_SPONSORING_FUTURE_RESERVES_NOT_SPONSORED | -1 | Source account is not sponsored. |
+
+## Revoke Sponsorship
+
+The logic of this operation depends on the state of the source account. 
+
+If the source account is not sponsored or is sponsored by the owner of the specified entry or sub-entry, then attempt to revoke the sponsorship. If the source account is sponsored, the next step depends on whether the entry is sponsored or not. If it is sponsored, attempt to transfer the sponsorship to the sponsor of the source account. If the entry is not sponsored, then establish the sponsorship. 
+
+Threshold: Medium
+
+Result: `RevokeSponsorshipResult`
+
+This operation is a union with two possible types -
+
+1. REVOKE_SPONSORSHIP_LEDGER_ENTRY
+
+| Parameters | Type | Description |
+| --- | --- | --- |
+| LedgerKey | ledgerKey | Ledger key that holds information to identify a specific ledgerEntry that may have it's sponsorship modified. Search for LedgerKey in [stellar-ledger-entries.x](https://github.com/stellar/stellar-core/blob/master/src/xdr/Stellar-ledger-entries.x) for more information.|
+
+2. REVOKE_SPONSORSHIP_SIGNER
```

Comment:
This `2` shows up as a `1` in the preview.

Reply:
Yeah, the Markdown renderer doesn't like lists being broken up like this unfortunately. You can see it [here](https://developers.stellar.org/docs/tutorials/send-and-receive-payments/#send-a-payment) as well in the step-by-step breakdown.